backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/snapshots/RemoveSnapshotSingleDiskLiveCommand.java

```diff
@@ -182,6 +182,13 @@ private RemoveSnapshotSingleDiskStep getInitialMergeStepForImage(Guid imageId) {
                 return RemoveSnapshotSingleDiskStep.DESTROY_IMAGE;
             }
         }
+        if (getImageInfoFromVdsm(getDestinationDiskImage()) == null) {
+            log.info("Image has been previously merged and deleted, proceeding to DB sync");
```

Comment:
I think this is not necessarily true, I'd change it to something like
```
log.warn("Image does not exist, attempting to synchronize the database")
```

Reply:
Ack